resource/vehicleProperties/client.lua

```diff
@@ -120,6 +122,9 @@ function lib.getVehicleProperties(vehicle)
         local colorPrimary, colorSecondary = GetVehicleColours(vehicle)
         local pearlescentColor, wheelColor = GetVehicleExtraColours(vehicle)
 
+        local paint1, p1_color, p1_pearlescentColor = GetVehicleModColor_1(vehicle)
+        local paint2, p1_color2 = GetVehicleModColor_2(vehicle)
```

Comment:
Unused variables with snake_case?

Reply:
Also should be `modColor1`, `modColor2`. Possibly need to store as an array if the other variables aren't trash?